tests/Common/SelectTest.php

```diff
@@ -712,7 +745,7 @@ public function testUnionAll()
     public function testAutobind()
     {
         // do these out of order
-        $this->query->having('baz IN (:baz)', ['baz' => ['dib', 'zim', 'gir']]);
+        $this->query->having('baz IN (?, ?, ?)', ['dib', 'zim', 'gir']);
```

Comment:
Why did you changed this test ? Can you add it as a separate test ? Same for below. So we can make sure nothing is breaking. So far I like the PR. Nice.

Reply:
Because this test was wrong and now it don't pass with my modification.
And there is a new test for my code : testWhereWithInlineArray

Reply:
@harikt I think @syrm is right. The test is probably wrong. Take a look here.

https://github.com/auraphp/Aura.SqlQuery/blob/091f470e6d8ebbaafd978a7984b231a4f8279ecc/tests/Common/SelectTest.php#L731

It is not possible to bind such an array.

Reply:
Awesome PR thanks @syrm 

I've prepared a refactored PR for this in #198.